libs/components/forms/src/index.ts

```diff
@@ -19,6 +19,7 @@ export { SkyFileDropChange } from './lib/modules/file-attachment/types/file-drop
 export { SkyFormErrorsModule } from './lib/modules/form-error/form-errors.module';
 export { SkyFormErrorModule } from './lib/modules/form-error/form-error.module';
 
+export { SkyInputBoxHintTextPipe } from './lib/modules/input-box/input-box-hint-text.pipe';
```

Comment:
Should we be exporting this as a lambda? We can still use it in templates when exported that way - but it ensures our consumers can't use the typescript type which wouldn't normally be a part of the public API.

Reply:
lambda26